packages/front-end/pages/report/[rid].tsx

```diff
@@ -49,6 +52,12 @@ export default function ReportPage() {
   const { data, error, mutate } = useApi<{ report: ReportInterface }>(
     `/report/${rid}`
   );
+  const { data: experimentData } = useApi<{
+    experiment: ExperimentInterfaceStringDates;
+    idea?: IdeaInterface;
+    visualChangesets: VisualChangesetInterface[];
+  }>(`/experiment/${data?.report.experimentId}`);
```

Comment:
When `data?.report.experimenttId` is not set, this will make a request to `/experiment/`, which isn't ideal.  For that case, you can pass `null` in as the URL path instead and the useApi hook should handle that correctly.

```ts
useApi(data?.report?.experimentId ? `/experiment/${data.report.experimentId}` : null)
```